### DISCUSSION TITLE
FinOps Hub ADF support SHIR option and also Private endpoint for storage accounts

### DISCUSSION THREAD

**Discussion by saleemmy** [microsoft/finops-toolkit#861] [Q&A] [answered]
I would like to configure the FInOps Hub to be self hosted integration runtime with the ADF and private end point for storage accounts.  Is it supported and possible to get all the features working with this configuration.

**Comment by flanakin** [comment]
Private endpoints aren't supported in hubs yet, but this is in the backlog. We actually have an open PR with this right now: https://github.com/microsoft/finops-toolkit/pull/609

In the meantime, you can absolutely configure private endpoints manually or even update the template to support private endpoints. Feel free to leverage what's in the PR in the meantime, if it's helpful. We are also open to more contributors participating in that PR, if you see anything that could be improved.

**Reply by saleemmy** [reply, marked as answer]
I have deployed PEP for both Data factory and storage account.  Also I have deployed the SHIR and integrated with the finops-hub data factory. It is all working fine collecting the data and displaying.
